cores/nRF5/Tone.cpp

```diff
@@ -74,7 +73,7 @@ class TonePwmConfig {
 };
 TonePwmConfig _pwm_config;
 
-inline static bool _is_pwm_enabled(NRF_PWM_Type const * pwm_instance) {
+static bool _is_pwm_enabled(NRF_PWM_Type const * pwm_instance) {
```

Comment:
inline is still OK, it is more like suggestion to gcc. It is up to gcc to decide e.g Ofast may inline the function, but Os may not since it increase the size. attribute always inline is more like directive to gcc to force it to do so. So yeah, it is normally not an issue on non timing-sensitive function. Anyway, I normally just put inline where I would like, and leave the decision to gcc.